monai/transforms/utility/dictionary.py

```diff
@@ -193,6 +193,26 @@ def __call__(self, data):
         return d
 
 
+class SimulateDelayd(MapTransform):
+    """
+    dictionary-based wrapper of :py:class:monai.transforms.utility.array.SimulateDelay.
+    """
+
+    def __init__(self, keys, delay_time=0.0):
+        """
```

Comment:
A suggestion here: Should we consider to support `delay_time` in **tuple** format that matches the keys?
Just like the `mode` in our RandAffined transform: https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/spatial/dictionary.py#L248
If `delay_time` is a single value, use it for all specified keys, if `delay_time` is a tuple, match every item of the specified keys.
For example, maybe we simulate 10 seconds for **image** loading delay and simulate 5 seconds for **label** loading delay, and 2 seconds for some **extra data** loading delay.
Thanks.